alacritty/src/display.rs

```diff
@@ -286,6 +287,21 @@ impl Display {
         size_info.cell_height = cell_height;
     }
 
+    /// Evict and replace glyph cache
+    pub fn handle_glyph_cache_refresh(&mut self, config: &Config) {
+        let (glyph_cache, ..) =
```

Comment:
I'm not sure if we've discussed this before, but we should handle the cell size changes too. We should not rely on the font size changing at the same time, instead we should either change the font size or evict the complete cache and create a new one. We shouldn't do both.

Reply:
Yeah I was going to carry over a lot of the cell size change work that I had included in the glyph cache refresh code prior to the rebase, but I thought it made more sense to trigger a `Resize` event for the sake of avoiding duplication of `handle_resize` code. I was kind of suspecting that you might ask me to rework that, but I also wasn't sure how you would feel about having the glyph cache refresh entirely separate from the `Resize` event either. Looking at your next comment, it seems like you might prefer them to be handled together anyway so I'm glad I didn't try to handle cell resizes here :)